src/Indigo/Contracts/HomebaseLite/Impl/Vote.hs

```diff
@@ -0,0 +1,50 @@
+-- SPDX-FileCopyrightText: 2022 Tezos Commons
+-- SPDX-License-Identifier: LicenseRef-MIT-TC
+
+{-# OPTIONS_GHC -Wno-redundant-constraints -Wno-orphans #-}
+
+module Indigo.Contracts.HomebaseLite.Impl.Vote
+  ( propose
+  , vote
+  ) where
+
+import Indigo hiding ((*))
+
+import Morley.Util.Interpolate (itu)
+
+propose :: IndigoEntrypoint tp
+propose _ = do
+  description [itu|
+    Called by a `participant` to submit a new proposal.
+    Takes as input an IPFS URI and a list of strings for the `n`  available choices.
+
+    - fails if the sender does not have a governance token amount of `minimum_balance` or more
+    - fails if a proposal for the same IPFS URI was already submitted before
+    - fails if the number of choices `n` is `0`
```

Comment:
I'm mentioning this here, but it applies to everywhere else too, since errors have docs, it should be unnecessary to explain these here and instead the error documentation should be:
- defined with the `errorDoc` quasiquote (as you already do)
- included automatically in the produced documentation both in each entrypoint that uses them as well as in a dedicated section

Reply:
FWIW, I found this pretty useful while implementing, so there's that.

Reply:
:thinking: I see, maybe I was too eager to suggest this and they could have been removed later on